configs/13.0/packages/gcc/stage_4

```diff
@@ -38,13 +38,16 @@ atcfg_configure() {
 		# Configure command for native builds
 		PATH=${at_dest}/bin:${PATH} \
 		CC="${at_dest}/bin/gcc" \
+		CXX="${at_dest}/bin/g++" \
+		AS="${at_dest}/bin/as" \
+		LD="${at_dest}/bin/ld" \
+		AR="${at_dest}/bin/ar" \
+		NM="${at_dest}/bin/nm" \
```

Comment:
For consistency's sake, could you also add RANLIB here, as you did for versions 11.0 and 12.0?